debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnectorConfig.java

```diff
@@ -405,6 +405,12 @@ else if (databaseNames != null) {
         }
 
         this.optionDatabaseCallbacks = config.getBoolean(DATABASE_CALLBACKS);
+        if (optionDatabaseCallbacks) {
+            LOGGER.warn("Database callbacks are enabled");
+        }
+        else {
+            LOGGER.warn("Database callbacks are disabled");
```

Comment:
Sorry for the incomplete original review but this one is "info". There's nothing wrong about callbacks being enabled or disabled to qualify this message for a warning.